### DISCUSSION TITLE
[URGENT] Routing and bounce management for Acelle mail

### DISCUSSION THREAD

**Discussion by khalidelmerrah** [postalserver/postal#2487] [Help with using Postal]
Hello team, thanks for the great you have done so far. 

i'm having issue configuring the routing section: 
- the domain is fully configured as you can see here : [https://ibb.co/Cbc7dZm](https://ibb.co/Cbc7dZm)
- the dns zone is valid : https://ibb.co/1TWtVj5
- the routing is created ( don't know if it's correct ) : [https://ibb.co/g4By2Rg](https://ibb.co/q5m0FBs) - [https://ibb.co/g4By2Rg](https://ibb.co/g4By2Rg)

my situation is i need bounce emails to be sent to this mailbox : bounce@logytalks.net
so that acelle mail can connect to it and get bounce data. 

ps: this is urgent.

**Comment by khalidelmerrah** [comment]
OP here : I tried sending emails to the address points but i'm not receiving anything on the incoming emails : 

<img src="https://i.ibb.co/grYHh7n/Screen-Shot-2023-06-08-at-1-00-10-PM.png" alt="Screen-Shot-2023-06-08-at-1-00-10-PM" border="0">

**Comment by willpower232** [comment]
If you're looking to catch bounces then you need to have a return path route as documented here https://docs.postalserver.io/other/auto-responders-and-bounces
